packages/types/src/article.ts

```diff
@@ -0,0 +1,36 @@
+import { z } from "zod"
+
+export const ArticleTagSchema = z.object({
+  name: z.string().min(1),
+})
+
+export type ArticleTagName = ArticleTag["name"]
+export type ArticleTag = z.infer<typeof ArticleTagSchema>
+
+export const ArticleTagWrite = ArticleTagSchema
+export type ArticleTagWrite = z.infer<typeof ArticleTagWrite>
+
+export const ArticleSchema = z.object({
+  id: z.string().ulid(),
+  createdAt: z.date(),
+  updatedAt: z.date(),
+  title: z.string().min(1),
+  author: z.string().min(1),
+  photographer: z.string().min(1),
```

Comment:
Could *technically* have a relation onto OwUser here, but I don't think todays use of this field really warrants it. It also makes the query for fetching the names more complex as we have to go through AWS Cognito

Reply:
In my opinion it's kind of unfortunate to not establish a relation to the given user for the author field. In the future we may want to do something like abakus does with profiles. At that point it makes sense to link to the profile of the author in the article. Like this: https://abakus.no/articles/488-weekly-uke-44. 

At least I would like a `select` for the author field in the write form. This way we could link them up in the future.